backend/src/models/payments.js

```diff
@@ -7,7 +7,7 @@ const model = db.define('payment_requests', {
     amount: Sequelize.DOUBLE,
     payment_status: {
         type: Sequelize.INTEGER,
-        defaultValue: 0,
+        defaultValue: -1,
```

Comment:
Have you checked with existing data that the default value is updated?. Or is it throwing any error?.